rosbag2_py/rosbag2_py/_info.pyi

```diff
@@ -2,7 +2,8 @@ import rosbag2_py._storage
 
 class Info:
     def __init__(self) -> None: ...
-    def print_output(self, arg0: rosbag2_py._storage.BagMetadata) -> None: ...
-    def print_output_topic_name_only(self, arg0: rosbag2_py._storage.BagMetadata) -> None: ...
-    def print_output_verbose(self, arg0: str, arg1: rosbag2_py._storage.BagMetadata) -> None: ...
+    def get_sorting_methods(self) -> Set[str]: ...
+    def print_output(self, arg0: rosbag2_py._storage.BagMetadata, arg1: str) -> None: ...
+    def print_output_topic_name_only(self, arg0: rosbag2_py._storage.BagMetadata, arg1: str) -> None: ...
+    def print_output_verbose(self, arg0: str, arg1: rosbag2_py._storage.BagMetadata, arg2: str) -> None: ...
```

Comment:
It looks like we are changing API here, which is publicly exposed. Likely we can't backport this PR.
@clalancette Could you please give a second opinion on that?

Reply:
@clalancette Thank you for the information I will take it into consideration in the backporting PR https://github.com/ros2/rosbag2/pull/1838.